index.html

```diff
@@ -0,0 +1,368 @@
+
+![](images/foldable-screen-device-laptop.png)
+*<br>A [=foldable-screen device=] as a laptop. ([Source](TODO))*
+
+![](images/foldable-screen-device-mobile.png)
+*<br>Another [=foldable-screen device=] as a mobile phone. ([Source](TODO))*
+
+### Multi-monitor setups
+
+A <dfn data-export="">multi-monitor setup</dfn> is a traditional mainstream setup in which a device (foldable or not) has one or more monitors connected to it (wired or not). Major desktop operating systems provide affordances to the user to arrange the logical position of the monitors as seen by the OS to match the physical location of the monitors as selected by the user. In this configuration, it is typical to mix and match monitors of different physical sizes in both landscape and horizontal configurations to optimize the screen estate for the usage model. Power users may also make use of advanced software tools to create virtual zones to which application windows can snap into (see e.g. [Microsoft PowerToys FancyZones](https://github.com/microsoft/PowerToys/wiki/FancyZones-Overview)). The virtual zones provide a similar abstraction to the [=window segments=].
+
+![](images/multi-monitor-setup.png)
+*<br>An example [=multi-monitor setup=] optimized for software development tasks with a [=traditional laptop=] connected to two [=monitors=] with one monitor in landscape orientation displaying code review side by side and one monitor in portrait orientation displaying a long change log listing. ([Source](TODO))*
+
+### Monitor
+
+A <dfn>monitor</dfn> is an output device that displays information in pictorial form. A monitor usually comprises the display, circuitry, casing, and power supply. ([Wikipedia](https://en.wikipedia.org/wiki/Computer_monitor))
+
+### Display
+
+A <dfn>display</dfn>, informally a [=screen=], is a display device for presentation of images, text, or video transmitted electronically, without producing a permanent record. Electronic visual displays include television sets, computer monitors, and digital signage. ([Wikipedia](https://en.wikipedia.org/wiki/Electronic_visual_display))
+
+### Screen
+
+<dfn>Screen</dfn> is a synonym to [=display=]. [=Screen=] is used more in software context (logical) while display is primarily used in hardware context (physical). See also the [=Screen interface=].
```

Comment:
How do we differentiate with:
Screen - the aggregate 2D space occupied by all the connected displays.
From https://github.com/webscreens/window-segments/blob/master/EXPLAINER.md

Reply:
We don't currently, but as part of this exercise need to. The tool flags the duplicate definitions, see the red pill at the top when in HTML Preview.

Reply:
Yeah, we need a term for the "overall visual workspace" of the device, whatever we choose to call the composition of multiple physical displays/screens.

Wikipedia says ["Multi-monitor, also called multi-display and multi-head, is the use of multiple physical display devices, such as monitors, televisions, and projectors, in order to increase the area available for computer programs running on a single computer system."](https://en.wikipedia.org/wiki/Multi-monitor)
CSSOM View's ["web-exposed-screen-information"](https://drafts.csswg.org/cssom-view/#web-exposed-screen-information) only uses singular output device terminology
MDN says ["The Screen interface represents a screen, usually the one on which the current window is being rendered... browsers determine which screen to report as current by detecting which screen has the center of the browser window."](https://developer.mozilla.org/en-US/docs/Web/API/Screen)
MSDN says ["The bounding rectangle of all the monitors is the _virtual screen_."](https://docs.microsoft.com/en-us/windows/win32/gdi/the-virtual-screen)
Apple says ["Maximize your workspace with extended desktop mode, which lets you enjoy full-screen apps and windows on each monitor"](https://support.apple.com/en-us/HT202351)
Ubuntu says ["With two screens, these display modes are available: Join Displays: screen edges are joined so things can pass from one display to another..."](https://help.ubuntu.com/stable/ubuntu-help/display-dual-monitors.html.en)
Arch Linux says ["Display refers to a collection of screens" and more...](https://wiki.archlinux.org/index.php/Multihead)
Chromebook Help says ["You can connect your Chromebook to a monitor or TV"](https://support.google.com/chromebook/answer/1060909?hl=en) :-/
Android mentions ["Multi-Display"](https://source.android.com/devices/tech/display/multi_display) and ["multi-screen"](https://source.android.com/devices/tech/display/multi_display/recommended-practices#foldable)
iOS/UIKit says ["The user can connect additional screens"](https://developer.apple.com/documentation/uikit/windows_and_screens/displaying_content_on_a_connected_screen)
Chromium code says ["A single Screen object exists regardless of the number of connected displays."](https://source.chromium.org/chromium/chromium/src/+/master:ui/display/screen.h;l=36)

Reply:
I was about to comment that Mike s comment on spanning using the word: "visual workspace" sounded good.

Reply:
"screen/visual workspace: the aggregate 2D space composed by all the active connected screens"
(some native English speaker might find more accurate wording)

Reply:
Added a placeholder for the "overall visual workspace" term (along with the very helpful references) in acda45b.